website/content/docs/job-specification/sidecar_task.mdx

```diff
@@ -156,6 +156,9 @@ meta.connect.sidecar_image = custom/envoy-${NOMAD_envoy_version}:latest
 - `volume_mount` <code>([VolumeMount][]: nil)</code> - Specifies where a group
   volume should be mounted.
 
+- `identity` <code>([Identity][]: nil)</code> - Expose [Workload Identity][] to
+  the task.
```

Comment:
This matches the documentation for `task.identity`, but we should probably include a little extra context here:


```suggestion
- `identity` <code>([Identity][]: nil)</code> - Expose an extra [Workload Identity][]
  to the task. Nomad automatically configures Consul Service Mesh sidecars with
  the required identities to communicate with Consul. Specifying this block lets
  you add extra identities that can be consumed by third-party services.
```